styleguide/_themes/derek/scss/patterns/announcements.scss

```diff
@@ -49,4 +57,8 @@
   .announcement-message {
     width: $full-width;
   }
+
+  .announcement-head {
+    text-align: center;
```

Comment:
Any thing we do on the mobile version, needs to be added to the subpanel version as well.  I realize announcements aren't currently working on zoolander inside the subpanels page, but basically all that needs to happen is your mobile code should also be placed here:

https://github.com/rackerlabs/zoolander/blob/master/styleguide/_themes/derek/scss/patterns/panels.scss#L14